### ISSUE TITLE
Across Homes/DataSet Neurals issue

### ISSUE THREAD

**Issue by Eno-Haicheng** [nilmtk/nilmtk-contrib#51]
Hello,

I'm new user of nilmtk, now i try to use seq2point neural network. I try to train (across home) the model of seq2point with the data of building 1 and building 2 of  dataset ukdale. But the value of loss did not improve from INF.  I try with redd dataset too, i got aways the same problem.

I'm not a expert on neural network, would it be possible to let me know how to solve this problem ?

## There is my code for seq2point :

experiment = {
  'power': {'mains': ['apparent','active'],'appliance': ['apparent','active']},
  'sample_rate': 1800,
  'appliances': ['fridge', 'dish washer','kettle','washer dryer'],
  'DROP_ALL_NANS': True,

  'methods': {"Seq2Point":Seq2Point({'n_epochs':50,'batch_size':1024})},   
  'train': {
    'datasets': {
      'UKDALE': {
        'path': 'C:\\Users\\HaichengLing\\Jupyter\\ukdale.h5',
        'buildings': {
              1: {
                'start_time': '2014-08-05',
                'end_time': '2014-12-30'
              }, 
              2: {
                'start_time': '2013-05-25',
                'end_time': '2013-07-15'
              }, 
            }
          },        
        }
      },    
  'test': {
    'datasets': {
        
        'UKDALE': {
            'path': 'C:\\Users\\HaichengLing\\Jupyter\\ukdale.h5',
            'buildings': {
                1: {
                    'start_time': '2014-12-30',
                    'end_time': '2015-01-03'
                    }
                }
            }
    
    },
        'metrics':['mae', 'rmse']
    }
}

## result of execution
Train on 11998 samples, validate on 2118 samples
Epoch 1/50
11998/11998 [==============================] - 5s 389us/step - loss: nan - val_loss: nan

Epoch 00001: val_loss did not improve from inf
Epoch 2/50
11998/11998 [==============================] - 6s 464us/step - loss: nan - val_loss: nan

Epoch 00002: val_loss did not improve from inf
Epoch 3/50
11998/11998 [==============================] - 6s 521us/step - loss: nan - val_loss: nan

Epoch 00003: val_loss did not improve from inf
Epoch 4/50
11998/11998 [==============================] - 6s 524us/step - loss: nan - val_loss: nan

Epoch 00004: val_loss did not improve from inf
Epoch 5/50
11998/11998 [==============================] - 6s 474us/step - loss: nan - val_loss: nan

Epoch 00005: val_loss did not improve from inf
Epoch 6/50
11998/11998 [==============================] - 6s 485us/step - loss: nan - val_loss: nan

Epoch 00006: val_loss did not improve from inf
Epoch 7/50
11998/11998 [==============================] - 6s 490us/step - loss: nan - val_loss: nan

Epoch 00007: val_loss did not improve from inf
Epoch 8/50
11998/11998 [==============================] - 6s 486us/step - loss: nan - val_loss: nan
